.gitignore

```diff
@@ -1,7 +1,12 @@
-/node_modules
+
+node_modules/
 npm-debug.log
 .DS_Store
 /*.env
 client/.env
 .env
 /.idea
+test.db
+globalConfig.json
+videos/
+screenshots/
```

Comment:
The cypress testing now saves video and screenshots. We do not need to commit these to source control. (see further down on the cypress section for more details.